src/navigations/root-stack.js

```diff
@@ -111,24 +112,24 @@ export const RootNavigator = ({currentScreen}) => {
         }}>
         <NetworkStatusIndicator hide={true} />
         {/* <StatusBar translucent backgroundColor="white" /> */}
-        <KeyboardAvoidingView style={{flex: 1}} enabled={Platform.OS === 'ios'} behavior="padding">
-          <RootStack.Navigator
-            screenOptions={{
-              headerShown: false,
-              safeAreaInsets: {
-                top: 0,
-                bottom: 0,
-                left: 0,
-                right: 0
-              }
-            }}>
-            {initialStartup.id !== null && initialStartup.id !== '' ? (
-              <RootStack.Screen name="AuthenticatedStack" component={AuthenticatedNavigator} />
-            ) : (
-              <RootStack.Screen name="UnauthenticatedStack" component={UnauthenticatedNavigator} />
-            )}
-          </RootStack.Navigator>
-        </KeyboardAvoidingView>
+        {/* <KeyboardAvoidingView style={{flex: 1}} enabled={Platform.OS === 'ios'} behavior="padding"> */}
+        <RootStack.Navigator
+          screenOptions={{
+            headerShown: false,
+            safeAreaInsets: {
+              top: 0,
+              bottom: 0,
+              left: 0,
+              right: 0
+            }
+          }}>
+          {initialStartup.id !== null && initialStartup.id !== '' ? (
+            <RootStack.Screen name="AuthenticatedStack" component={AuthenticatedNavigator} />
+          ) : (
+            <RootStack.Screen name="UnauthenticatedStack" component={UnauthenticatedNavigator} />
+          )}
+        </RootStack.Navigator>
+        {/* </KeyboardAvoidingView> */}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `KeyboardAvoidingView` and its associated props might cause issues with keyboard overlap on iOS devices. If the new `KeyboardWrapper` component is meant to replace this functionality, it should be used here as well.

```diff
-        {/* <KeyboardAvoidingView style={{flex: 1}} enabled={Platform.OS === 'ios'} behavior="padding"> */}
+        <KeyboardWrapper>
           <RootStack.Navigator
             screenOptions={{
               headerShown: false,
               safeAreaInsets: {
                 top: 0,
                 bottom: 0,
                 left: 0,
                 right: 0
               }
             }}>
             {initialStartup.id !== null && initialStartup.id !== '' ? (
               <RootStack.Screen name="AuthenticatedStack" component={AuthenticatedNavigator} />
             ) : (
               <RootStack.Screen name="UnauthenticatedStack" component={UnauthenticatedNavigator} />
             )}
           </RootStack.Navigator>
-        {/* </KeyboardAvoidingView> */}
+        </KeyboardWrapper>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->